### ISSUE TITLE
Feature request: `pipenv install --index myindex mypackage`

### ISSUE THREAD

**Issue by patrick-jones** [pypa/pipenv#1852]
I would like to request a command line option for specifying the source during `pip install`. For example, given this Pipfile (from [the docs](https://docs.pipenv.org/advanced/#specifying-package-indexes)):

```
[[source]]
url = "https://pypi.python.org/simple"
verify_ssl = true
name = "pypi"

[[source]]
url = "http://pypi.home.kennethreitz.org/simple"
verify_ssl = false
name = "home"
```

I would like to be able to invoke something like:

`pipenv install --index home mypackage`

Which would install from my "home" index and add this into my Pipfile:
```
[packages]
mypackage = {version="*", index="home"}
```

**Comment by techalchemy** [comment]
Makes sense, although if it's a package that isn't on pypi and your index is in your pipfile already it may work just fine to leave the index off

**Comment by uranusjr** [comment]
Is this technically possible though? While there is `--extra-index-url` in pip, I don’t think you can specify which index to use for each dependency, only the installation as a whole.

**Comment by jakul** [comment]
The example at https://docs.pipenv.org/advanced/#specifying-package-indexes specifies a different index for each requirement, so Pipenv must already support it.

**Comment by techalchemy** [comment]
Yeah good point, doesn’t this already work?

**Comment by techalchemy** [comment]
Try `pip install -e git+https://github.com/pypa/pipenv.git@1973-extra-index-urls#egg=pipenv` and let me know if it works
